rootfs/mkrootfs_debian.sh

```diff
@@ -152,18 +152,18 @@ packages=(
 	strace
 	zlib1g
 )
-packages=$(IFS=, && echo "${packages[*]}")
```

Comment:
NIt: In my opinion this rename is confusing. What is the complaint and can we disable it instead?

Reply:
Sorry, the highlight is weird. I am referring to the new name, `packages2`, not the old name.